<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stegano Tool</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 11pt &quot;MathJax_Caligraphic&quot;;
color:rgb(193, 125, 17);
font-size: 30px</string>
    </property>
    <property name="text">
     <string>Secret Steganography Tool V 1.0</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>761</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>424</y>
       <width>230</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid;
border-color: rgb(193, 125, 17);
border-radius: 3px</string>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(193, 125, 17);
border-radius: 3px</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Stego Options&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>100</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	border: 1px solid;
	border-color: rgb(193, 125, 17);
	border-radius: 3px
}
QPushButton:hover{
	color: rgb(245, 121, 0)
}</string>
      </property>
      <property name="text">
       <string>     Hide</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>100</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	border: 1px solid;
	border-color: rgb(193, 125, 17);
	border-radius: 3px
}
QPushButton:hover{
	color: rgb(245, 121, 0)
}
</string>
      </property>
      <property name="text">
       <string>    Show</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>231</width>
       <height>89</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid;
border-color: rgb(193, 125, 17);
border-radius: 3px</string>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(193, 125, 17);
border-radius: 3px</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Image Options&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>454</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/photos/hide.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>454</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/photos/eye.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>170</width>
       <height>39</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
	color: rgb(255, 255, 255);
	border: 1px solid;
	border-color: rgb(193, 125, 17);
	border-radius: 3px
}
QPushButton:hover{
	color: rgb(245, 121, 0)
}</string>
     </property>
     <property name="text">
      <string>      Select Image</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>40</width>
       <height>29</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/photos/folder.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>231</width>
       <height>140</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid;
border-color: rgb(193, 125, 17);
border-radius: 3px</string>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(193, 125, 17);
border-radius: 3px</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt; Encryption&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>30</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/photos/key-tool.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>210</width>
       <height>69</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
border: 1px solid rgb(46, 52, 54);
border-radius: 2px</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>40</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
border-left: 3px solid rgb(85, 87, 83)</string>
     </property>
     <property name="text">
      <string> Key:</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>230</width>
       <height>129</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid;
border-color: rgb(193, 125, 17);
border-radius: 3px;
color: rgb(255, 255, 255)</string>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(193, 125, 17);
border-radius: 3px</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;your message&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>210</width>
       <height>89</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
border: 1px solid rgb(46, 52, 54);
border-radius: 2px</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>30</y>
       <width>20</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/photos/one.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>132</y>
       <width>20</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/photos/two.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>281</y>
       <width>20</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/photos/three.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>425</y>
       <width>20</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/photos/four.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>25</y>
       <width>480</width>
       <height>480</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(115, 210, 22);</string>
     </property>
     <property name="title">
      <string>Informations and Progress</string>
     </property>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>40</y>
        <width>410</width>
        <height>160</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(252, 233, 79);</string>
      </property>
      <property name="title">
       <string>Image Info :</string>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>121</y>
         <width>60</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(117, 80, 123);
border-radius: 2px</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Size :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>192</x>
         <y>121</y>
         <width>90</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(117, 80, 123);
border-radius: 2px</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Extension :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(117, 80, 123);
border-radius: 2px</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Name :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(117, 80, 123);
border-radius: 2px</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Path :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>117</y>
        <width>310</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(117, 80, 123);
border-radius: 2px;
color: rgb(255, 255, 255)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>77</y>
        <width>310</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(117, 80, 123);
border-radius: 2px;
color: rgb(255, 255, 255)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>158</y>
        <width>90</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(117, 80, 123);
border-radius: 2px;
color: rgb(255, 255, 255)</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>209</y>
        <width>410</width>
        <height>160</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(252, 233, 79);</string>
      </property>
      <property name="title">
       <string>Process Info :</string>
      </property>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>79</y>
         <width>60</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(117, 80, 123);
border-radius: 2px</string>
       </property>
       <property name="text">
        <string> Output :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>42</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(117, 80, 123);
border-radius: 2px</string>
       </property>
       <property name="text">
        <string> Operation :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>43</y>
         <width>130</width>
         <height>18</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(239, 41, 41);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>111</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {border: 1px solid rgb(136, 138, 133);border-radius: 3px}
QPushButton:hover {
	border: 1px solid rgb(252, 175, 62)
}</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>285</y>
        <width>240</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(117, 80, 123);
border-radius: 2px;
color: rgb(255, 255, 255)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>380</y>
        <width>420</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Karumbi</family>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;By AlGoRithM&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>158</y>
        <width>100</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(117, 80, 123);
border-radius: 2px;
color: rgb(255, 255, 255)</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>307</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
	color: rgb(255, 255, 255);
	border: 1px solid;
	border-color: rgb(193, 125, 17);
	border-radius: 3px
}
QPushButton:hover{
	color: rgb(245, 121, 0)
}</string>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <zorder>groupBox_5</zorder>
    <zorder>groupBox_3</zorder>
    <zorder>groupBox_2</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>label_2</zorder>
    <zorder>groupBox_4</zorder>
    <zorder>label_8</zorder>
    <zorder>label_3</zorder>
    <zorder>plainTextEdit</zorder>
    <zorder>label_9</zorder>
    <zorder>label_10</zorder>
    <zorder>plainTextEdit_2</zorder>
    <zorder>label_20</zorder>
    <zorder>label_21</zorder>
    <zorder>label_22</zorder>
    <zorder>label_23</zorder>
    <zorder>groupBox_6</zorder>
    <zorder>pushButton_5</zorder>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="photos.qrc"/>
 </resources>
 <connections/>
</ui>
